### PR TITLE
Prototype code (conversion errors)

### PR DESCRIPTION
**Pull request by ananyakidambi**

### CODE REVIEW

**Review by jmullins16 — CHANGES_REQUESTED** [review]
Glad you got this checked in. 
1) Please merge from main back into your branch. This should eliminate simgui*.json files from being tracked for changes, and the auto-formatter should be enabled.
2) It is difficult to review your changes, because the entire ExampleSmartMotorController.java file (all lines) as changed. I know this isn't true, because see many lines are the same. I suspect this might be because you're on WIndows (which can change the line endings on EVERY line if incorrectly configured). We'll look at this more closely next time your in.

File: Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java
Comment (grace-mills) [inline Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java:1]: This needs to be set to true to allow limelight to work and recognize aprilTags. If it is effecting your motor controller maybe there is a work around

File: Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java [hunk 1]
```diff
@@ -208,10 +206,6 @@ public void updateSwerveOdometry( )
 
   private void updateSmartDashboard( )
```
Comment (grace-mills) [inline Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java:207]: These are also SmartDashboard outputs that give the robots estimated position and are used with limelight

**Review by hpark-24 — CHANGES_REQUESTED** [review]
It looks like a lot of code is being deleted here that I don't think you meant to delete (i.e. poseEstimation in Comp2023-Imported). Could you please make sure that only your changes are in this pull request?

**Review by jmullins16 — CHANGES_REQUESTED** [review]
It appears that your branch has been corrupted a bit. Looks like it has deleted lots of files that should not have been affected in other projects. We need to recover just your changes and do this again before merging.

**Pull request closed without merging**